filer/private/sass/components/_image-info.scss

```diff
@@ -34,6 +34,10 @@
         margin-bottom: 20px;
         padding: 25px;
         border-radius: 0;
+        // fixes object tools placement on image detail page
+        + #content-main .object-tools {
+            top: -310px;
```

Comment:
does that also work on popups and other places, that magic number seems a bit weird.